### DISCUSSION TITLE
Visitor map

### DISCUSSION THREAD

**Discussion by CaucherWang** [academicpages/academicpages.github.io#2056] [Ideas]
A visitor map includes a world map, and the locations where visitors access our pages. It can also count how many times of access sent from the locations.
It shows the influence of our pages.

**Comment by mendax1234** [comment]
> A visitor map includes a world map, and the locations where visitors access our pages. It can also count how many times of access sent from the locations. It shows the influence of our pages.

I believe using clustrmap is a bestway. You can see the [my website](https://mendax1234.github.io/) for reference.

But I have also created an [issue](https://github.com/academicpages/academicpages.github.io/issues/2101) related to it if you want to add the visitormap in the footer.
